### ISSUE TITLE
Create splash page for project

### ISSUE THREAD

**Issue by CliffordAnderson** [HeardLibrary/twine-for-wikicite#69]
We need a splash page that links to the Twine language versions in a cleaner/clearer way

**Comment by baskaufs** [comment]
We should consider just having the landing URL go to a splash page (in Twine) if no language is specified. We could then use the same theme that's already built. I can play with this to come up with a test.

**Comment by baskaufs** [comment]
@CliffordAnderson @cylew @Talinum @awesolek2 @marjans74 OK, check out this behavior:
<https://heardlibrary.github.io/twine-for-wikicite/wikicite4librarians>
don't worry about the exact content/styling, just the behavior for now.

Here's what happens. If someone uses the generic link above, the Twine recognizes that the language is unknown and presents the user with buttons to chose a language. When they click the button, the chosen language is set and the user is sent to the splash page in their language, which includes the permalink for their language. 

If the user uses a language specific URL like:
<https://heardlibrary.github.io/twine-for-wikicite/wikicite4librarians?en>
<https://heardlibrary.github.io/twine-for-wikicite/wikicite4librarians?es>
<https://heardlibrary.github.io/twine-for-wikicite/wikicite4librarians?zh-Hans>
the user goes directly to the splash page without having to choose a language. I haven't fixed it yet, but I could rig the script so that if they come directly without choosing a language, the permalink text would be hidden since they presumably already know what it is.

If this flow/approach is satisfactory, I can clean up the formatting/coding and we can decide what we actually want to have on the splash page. Please let me know.

**Comment by Talinum** [comment]
If you're in a video and you keep hitting the back button, it takes you back to the language choice page, which I think is good.
👍👍

**Comment by baskaufs** [comment]
@Talinum I can't replicate that. For me in Firefox, the browser back button takes me to the previous page outside of the Twine (or does nothing if I went there by a direct link). The back arrow on the navigation pane at the left takes me back to the previous Twine passage that I had been on. I think these are expected behaviors -- we probably need to tell people on the splash page that they should navigate with the navigation pane buttons and not the browser buttons. 

Were you watching the videos embedded or did you click through and watch them on YouTube? The behavior may also be different on Chrome or Safari -- I didn't try them.

**Comment by marjans74** [comment]
Behavior is same in Chrome.  If I select to watch on YouTube the only option is browser back button.  To me, that would seem evident, no explanation needed.   If we're going to mention not using browser back button, if not watching YouTube, do we want to also mention this?

**Comment by baskaufs** [comment]
When I do Watch on YouTube, it opens in a different tab, so that isn't an issue. When you go back, does it restart the Twine to the splash page, or does it return you to the page you left off on. If the latter, then we are fine. If it starts you over, then we probably need to warn people that leaving the site may make it difficult to get back to where they left off. The same would be true if they clicked on any of the TLDR or further info links if it doesn't return them to where they left off. Does it return them to the same passage if they click the back button? On my browser, it returns you to where you left off.

**Comment by marjans74** [comment]
Back returns me to page I left off on.  Same for TLDR.

**Comment by baskaufs** [comment]
Copied from https://github.com/HeardLibrary/twine-for-wikicite/issues/68: 

> Whenever we decide on this (domain name), I will change the name of the Twine story to it. That causes it to appear at the top of the navigation page. For the new test splash page version, I did this, so you see "wikicite4librarians" at the top of the nav page (instead of "Learn Wikidata", which is on the current index.html).
> 
> Also, on the splash page, we need some kind of reference to WikiCite. If rando people discover it, they might be scratching their heads since most of the content is generic enough that people wouldn't necessarily pick up that this if focused on librarians working in the arena of WikiCite.
> 
> See also #70 where I raised the issue of where we should link to our grant proposal and to our GitHub site. At the bottom of the splash? On the credits page? At the bottom of the nav page?

**Comment by baskaufs** [comment]
@CliffordAnderson is https://github.com/HeardLibrary/twine-for-wikicite/issues/69#issuecomment-858614059 feasible enough that I should work a bit more on the formatting and coding to make the permalinks appear/disappear? We will still need to figure out what text and links need to be there. It would be a quick solution that would maintain our site theme and avoid us having to figure out how to manage a separate web page.

**Comment by CliffordAnderson** [comment]
@baskaufs, it's certainly feasible. I just want to makre sure that we can make the landing page stylish and attractive. But I agree that's preferable to keep with the Twine site than manage a separate web page, though the latter would be easier to style.

**Comment by baskaufs** [comment]
OK, I won't do any more work on this until a decision is made.

**Comment by baskaufs** [comment]
I have now set the index.html page to the version that opens to a landing page rather than the first lesson. I have rigged it so that if no language-specific URL is used, the language selection buttons show up. After one is chosen, the page provides a permalink to the language-specific page for that language.

If one arrives at the site through a language-specific URL, no language choice buttons are displayed and the permalink is not shown because the user presumably already knew about it, since that link was used to arrive at the page.

I realize this is probably not fancy enough for what we will want eventually, but it is an improvement over the previous landing page. @cylew @Talinum  I do still need the Spanish and Chinese translations for "Permalink to this page:" and "get started"
